release-engineering/role-handbooks/branch-manager.md

```diff
@@ -324,6 +330,12 @@ The command example below is to stage a build for a beta release:
 ./gcbmgr stage release-1.16 --buildversion=v1.16.0-alpha.3.7+b3bde1fe32dd4a --nomock
 ```
 
+or
+
+```shell
+./gcbmgr stage release-1.16 --buildversion=$(curl -Ls https://dl.k8s.io/ci/latest-1.16.txt) --nomock
```

Comment:
This will fail for a `beta.0` job because the build job that produces the next `latest-x.y.txt` version marker does not exist yet.

Reply:
fixed

Reply:
@justaugustus PTAL